bin/rtspin.c

```diff
@@ -215,7 +215,7 @@ static int loop_for(double exec_time, double emergency_exit)
 	int tmp = 0;
 
     if (cycles_ms) {
-        int count = cycles_ms * (int)(exec_time * 1000);
+        int count = cycles_ms * (int)(exec_time * 1000.0d + 0.5d);
```

Comment:
I'm not sure this change is needed. Since `exec_time` is already a `double`, isn't the constant `1000`  implicitly casted to `double` before the multiplication anyway?

And why the `+ 0.5d`?

Reply:
> Since exec_time is already a double, isn't the constant 1000 implicitly cast to double before the multiplication anyway?

I guess it left from my debugging. It is not really necessary.

> And why the + 0.5d?
It is for rounding it up to the nearest bigger integer. Needed because of <double> representation issue.